src/background/certificate/providers/ServerProvider.ts

```diff
@@ -3,13 +3,28 @@ import { WebRequest } from "webextension-polyfill-ts";
 import { APIResponseBody } from "../../../shared/types/api/APIResponseBody";
 import { RawCertificate } from "../../../shared/types/certificate/RawCertificate";
 import { CodedError } from "../../../shared/types/errors/CodedError";
+import { ServerUnavailableError } from "../../../types/errors/ServerUnavailableError";
 import { ErrorFactory } from "../factories/ErrorFactory";
 import { CertificateProvider } from "./CertificateProvider";
 
-const SERVER_URL =
-  "https://snonitze65.execute-api.eu-central-1.amazonaws.com/getCertificate/";
-
 export class ServerProvider implements CertificateProvider {
+  static readonly defaultServerUrl =
+    "https://snonitze65.execute-api.eu-central-1.amazonaws.com/";
+  static readonly endpoint = "getCertificate/";
```

Comment:
See https://github.com/Chexxo/Server/pull/36
```suggestion
  static readonly endpoint = "certificate/";
```